src/SdkCommon/CR.test.reference.props

```diff
@@ -1,7 +1,11 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-	<Import Project="$([MSBuild]::GetPathOfFileAbove('test.props'))" />	
+	<!-- <Import Project="$([MSBuild]::GetPathOfFileAbove('test.props'))" />	 -->
```

Comment:
Should we delete this instead of commenting it out?

Reply:
Yes we should delete it. During testing things were not working, so I thought commenting, but now that everything works, this will be deleted.